tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -376,8 +376,34 @@ public async Task AcquireTokenWithClientSecretFromAdfsAsync()
                                             .WithClientSecret(secret.Value)
                                             .BuildConcrete();
 
-            //AuthenticationResult authResult = await msalConfidentialClient.AcquireTokenForClientAsync(AdfsScopes).ConfigureAwait(false);
-            AuthenticationResult authResult = await msalConfidentialClient.AcquireTokenForClient(s_adfsScopes).ExecuteAsync().ConfigureAwait(false);
+            AuthenticationResult authResult = await msalConfidentialClient
+                .AcquireTokenForClient(s_adfsScopes)
+                .ExecuteAsync()
+                .ConfigureAwait(false);
+
+            Assert.IsNotNull(authResult);
+            Assert.IsNotNull(authResult.AccessToken);
+            Assert.IsNull(authResult.IdToken);
+        }
+
+        [TestMethod]
+        [TestCategory(TestCategories.ADFS)]
```

Comment:
There are other ADFS tests that do not have a category. should probably be added

Reply:
Done.